src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -367,6 +378,16 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
       value: 'BEN',
       SHORT_DESC: 'BC Benefit Company',
       text: 'BC Benefit Company'
+    },
+    {
+      value: 'ULC',
+      SHORT_DESC: 'BC Unlimited Liability Company',
+      text: 'BC Unlimited Liability Company'
+    },
+    {
+      value: 'CC',
+      SHORT_DESC: 'BC Community Contribution Company',
+      text: 'BC Community Contribution Company'
```

Comment:
dropdown options included in src/resources/Alteration/BEN.ts, can be removed here.